cli/src/main/java/org/aya/cli/repl/jline/completer/CmdCompleter.java

```diff
@@ -36,10 +36,9 @@ public void complete(LineReader reader, ParsedLine line, List<Candidate> candida
     }
     var trim = line.line().trim();
     if (trim.startsWith(Command.PREFIX)) {
-      // TODO: replace with Option.mapNotNull
-      cmd.parse(trim.substring(1)).command()
-        .flatMap(c -> Option.of(c.argFactory()))
-        .flatMap(arg -> Option.of(arg.completer()))
+      cmd.parse(trim.substring(1)).command().view()
+        .mapNotNull(CommandManager.CommandGen::argFactory)
+        .mapNotNull(CommandArg::completer)
```

Comment:
I think we should check there's only one completer.

Reply:
Why? For several completers we just join all of them

Reply:
Well ok. I think it's fine now.

Reply:
My head is temporarily offline